tests/test_pdf.py

```diff
@@ -32,6 +32,13 @@ def test_pdf_page_count_src_pdf_not_a_pdf():
         pdf_page_count(BytesIO(file_data))
 
 
+@pytest.mark.parametrize("page_count, expected_result", [
+    (None, False), (3, False), (11, True)
```

Comment:
Always best to test the boundary cases to make sure you haven’t made an off-by-one error:
```suggestion
    (None, False), (10, False), (11, True)
```

Reply:
Good idea, done ☺️